blockchains/verge/info/info.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Verge",
+    "website": "https://vergecurrency.com",
+    "description": "Verge is a privacy-focused cryptocurrency and blockchain that seeks to offer a fast, efficient, decentralized payments network that improves upon the original Bitcoin (BTC) blockchain.",
+    "explorer": "https://verge-blockchain.info",
+    "symbol": "XVG",
+    "type": "coin",
+    "decimals": 8,
```

Comment:
Verge seems to have 6 decimals: https://github.com/vergecurrency/verge/blob/b9a3f7278ba9c71e993decf38e5fcf731feb8c1d/src/qt/vergeunits.cpp#L89
https://github.com/trustwallet/wallet-core/blob/4443c337f6337aa7fcb76bceb3987ad6a1fbbaf2/registry.json#L444